packages/main/src/Table.hbs

```diff
@@ -1,7 +1,7 @@
-<table border="0" cellspacing="0" cellpadding="0">
+<table border="0" cellspacing="0" cellpadding="0" @keydown="{{_onkeydown}}">
```

Comment:
Hello @Martotko 
please also remove the `_eventHandlersByConvention: true` from the component metadata as not required any more.

Reply:
Done.